BHoM_Adapter/BHoM_Adapter.csproj

```
@@ -69,6 +69,9 @@
       <Private>false</Private>
       <SpecificVersion>false</SpecificVersion>
     </Reference>
+    <Reference Include="Reflection_Engine">
+      <HintPath>..\..\..\..\..\..\ProgramData\BHoM\Assemblies\Reflection_Engine.dll</HintPath>
```

```suggestion
      <HintPath>C:\ProgramData\BHoM\Assemblies\Reflection_Engine.dll</HintPath>
```

You'll also need `SpecificVersion` and `CopyLocal` to be false - best to let the bot pick that up